<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Led Array Manager V1.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Drawing Sketch</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="pushButton_5">
            <property name="text">
             <string>Led</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_4">
            <property name="text">
             <string>Cursor</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_6">
            <property name="text">
             <string>Eraser</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGraphicsView" name="sketchgraphicsView"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Fancy</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Cross</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>Square</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNew">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuAppearance">
     <property name="title">
      <string>Appearance</string>
     </property>
     <addaction name="actionDark_Theme"/>
     <addaction name="actionWhite_Theme"/>
     <addaction name="actionGreen_Theme"/>
     <addaction name="actionBlue_Theme"/>
     <addaction name="actionYellow_Theme"/>
    </widget>
    <addaction name="menuAppearance"/>
   </widget>
   <addaction name="menuNew"/>
   <addaction name="menuSettings"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionDark_Theme">
   <property name="text">
    <string>Dark Theme</string>
   </property>
  </action>
  <action name="actionWhite_Theme">
   <property name="text">
    <string>White Theme</string>
   </property>
  </action>
  <action name="actionGreen_Theme">
   <property name="text">
    <string>Green Theme</string>
   </property>
  </action>
  <action name="actionBlue_Theme">
   <property name="text">
    <string>Blue Theme</string>
   </property>
  </action>
  <action name="actionYellow_Theme">
   <property name="text">
    <string>Yellow Theme</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
